packages/nimble-components/src/anchor/template.ts

```diff
@@ -0,0 +1,56 @@
+import { html, ref, slotted, ViewTemplate } from '@microsoft/fast-element';
+import type {
+    AnchorOptions,
+    FoundationElementTemplate
+} from '@microsoft/fast-foundation';
+import type { Anchor } from '.';
+
+// prettier-ignore
+export const template: FoundationElementTemplate<
+ViewTemplate<Anchor>,
+AnchorOptions
+> = () => html<Anchor>`
+    <a
+        class="control"
+        part="control"
+        download="${x => x.download}"
+        href=${x => x.href}
+        hreflang="${x => x.hreflang}"
+        ping="${x => x.ping}"
+        referrerpolicy="${x => x.referrerpolicy}"
+        rel="${x => x.rel}"
+        target="${x => x.target}"
+        type="${x => x.type}"
+        aria-atomic="${x => x.ariaAtomic}"
+        aria-busy="${x => x.ariaBusy}"
+        aria-controls="${x => x.ariaControls}"
+        aria-current="${x => x.ariaCurrent}"
+        aria-describedby="${x => x.ariaDescribedby}"
+        aria-details="${x => x.ariaDetails}"
+        aria-disabled="${x => x.ariaDisabled}"
+        aria-errormessage="${x => x.ariaErrormessage}"
+        aria-expanded="${x => x.ariaExpanded}"
+        aria-flowto="${x => x.ariaFlowto}"
+        aria-haspopup="${x => x.ariaHaspopup}"
+        aria-hidden="${x => x.ariaHidden}"
+        aria-invalid="${x => x.ariaInvalid}"
+        aria-keyshortcuts="${x => x.ariaKeyshortcuts}"
+        aria-label="${x => x.ariaLabel}"
+        aria-labelledby="${x => x.ariaLabelledby}"
+        aria-live="${x => x.ariaLive}"
+        aria-owns="${x => x.ariaOwns}"
+        aria-relevant="${x => x.ariaRelevant}"
+        aria-roledescription="${x => x.ariaRoledescription}"
+        ${ref('control')}
+    >
+        <span part="start">
+            <slot name="start">
+            </slot>
+        </span>
```

Comment:
If we are inlining these templates to fulfill the type then we should make sure to include the ref bindings since they are in the type: https://github.com/microsoft/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/patterns/start-end.ts#L126

Reply:
Added